src/features/court/components/sub-court/index.module.scss

```diff
@@ -0,0 +1,21 @@
+.container {
+  height: 100%;
+  width: 100%;
+}
+
+.court_image {
```

Comment:
@kavimaluskam how do you usually name the css class? Which format you use?

Reply:
i use `-` over `_`.

for naming ideas, usually its like the react module itself (i.e. `HalfCourt`) or
some styling with the module (`GreenCourt`, `FatDom`) ?

Reply:
http://getbem.com/naming/
i think we should follow bem naming for css.

A blog post with more examples
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

Reply:
<img width="505" alt="Screenshot 2020-12-30 at 19 30 13" src="https://user-images.githubusercontent.com/21273855/103373563-90a2a600-4ad5-11eb-9c4f-1187eebd3e6f.png">
I think the lib is converting scss into the naming pattern. Also, I couldn't import and have variable `styles.sub-court`.

@kavimaluskam check the update make sense to you?

Reply:
I wonder the name would be too generic 🤔 , but it looks OK as long as the naming is only used in the `sub-court` or `half-court` module.

Reply:
It follows my naming practice as in styled-component. 

But not sure how to do when we have some global css/class naming needs.

Reply:
> I wonder the name would be too generic

I think that's why it's put the block part name in front and then the sub name